gladier_kanzus/tools/dials_stills.py

```diff
@@ -1,37 +1,42 @@
 from gladier import GladierBaseTool, generate_flow_definition
 
-def stills_process(**data):
+def dials_stills(**data):
     import os
     import subprocess
-    from subprocess import PIPE
 
-    
     proc_dir = data['proc_dir']
     data_dir = data['data_dir']
-    input_files = data['input_files']
-    run_num = data['input_files'].split("_")[-2]
-    
-    if 'suffix' in data:
-        phil_name = f"{proc_dir}/process_{run_num}_{data['suffix']}.phil"
-    else:
-        phil_name = f"{proc_dir}/process_{run_num}.phil"
+    run_num = data['run_num']
+    chip_name = data['chip_name']
+    cbf_num = data['cbf_num']
+    batch_size = data['stills_batch_size']
 
-    file_end = data['input_range'].split("..")[-1]
-  
-    if not "timeout" in data:
-        data["timeout"] = 1200
+    phil_name = f"{proc_dir}/process_{run_num}.phil"
 
-    dials_path = data.get('dials_path','')
-    cmd = f'source {dials_path}/dials_env.sh && timeout {data["timeout"]} dials.stills_process {phil_name} {data_dir}/{input_files} > log-{file_end}.txt'
+    cbf_start = cbf_num - batch_size
+    cbf_end = cbf_num - 1
+
+    file_start = f"{chip_name}_{run_num}_{cbf_start}.cbf"
+    file_end = f"{chip_name}_{run_num}_{cbf_end}.cbf"
 
+    input_files = f"{chip_name}_{run_num}_{{{str(cbf_start).zfill(5)}..{str(cbf_end).zfill(5)}}}.cbf"
+
+    timeout = data.get('timeout', 1200)
+
+    logname = 'log-' + data['filename'].replace('.cbf','')
     
-    os.chdir(proc_dir) ##Need to guarantee the worker is at the correct location..
-    res = subprocess.run(cmd, stdout=PIPE, stderr=PIPE,
+    dials_path = data.get('dials_path','')
+    cmd = f'source {dials_path}/dials_env.sh && timeout {timeout} dials.stills_process {phil_name} {data_dir}/{input_files} > {logname}.txt'
```

Comment:
Can you add a comment on why it's sourcing dials_env.sh?

Reply:
Also, this line is redirecting output to `logname`, is that intentional instead of returning res.stdout?

Reply:
For the return value below, an alternative to returnig the output -- which will probably now be nil -- would be returning the full path of the log so users can investigate. Returning `res.stderr` would be good to if that produces anything interesting.